### PR TITLE
GHA workflows: Streamline workflows to remove a couple of unnecssary steps

### PR DESCRIPTION
**Pull request by tim-eves**

### CODE REVIEW

File: .github/workflows/staging.yml [hunk 1]
```diff
@@ -30,23 +30,17 @@ jobs:
...
-    - uses: actions/checkout@v2
-      with:
-        repository: silnrsi/sldrtools
-        path: tools
-    
     # Install SLDR tools
     - name: Setup sldrtools
       working-directory: tools
-      run: pip install .
+      run: pip install 'git+https://github.com/silnrsi/sldrtools'
```
Comment (alerque) [inline .github/workflows/staging.yml:36]: Why hard code the URL here? This will prohibit the workflow from working for branches and forks.

**Review by alerque — CHANGES_REQUESTED** [review]
This PR does not do what it says on the tin (it is not a typo fix) and it will cause other side effects like making the CI job miss-represent the situation when run from branches, PRs, and forks.

File: .github/workflows/staging.yml (hunk 1 above)
Comment (alerque) [reply .github/workflows/staging.yml:36]: Nevermind I see you're using tooling from one repo to manipulate a different one, sorry I missed that.

**Review by alerque — APPROVED** [review]

File: .github/workflows/staging.yml (hunk 1 above)
Comment (tim-eves) [reply .github/workflows/staging.yml:35]: Sorry I have no idea how you got added as a review this is an internal development PR to test everything works before merging.

**Pull request merged**